src/deploy/osp_deployer/profiles/csp.json

```diff
@@ -11,7 +11,16 @@
                   "1GB"
                ]
             },
-            "numa_enable": ""
+            "numa_enable": "",
+	    "numa_hostos_cpu_count": {
```

Comment:
Can you fix the indenting here?  The above line should line up with the line above it.

Reply:
Can you also copy the numa_hostos_cpy_count section into the xsp profile so that the values for it are correctly restricted there as well?

Reply:
Fixed
Required added to xsp.json